create-environment/action.yml

```diff
@@ -4,12 +4,24 @@ name: 'create-environment'
 description: 'Power Platform Admin Create Environment'
 inputs:
   user-name:
-    description: 'Power Platform user name, e.g. myname@my-org.onmicrosoft.com'
-    required: true
+    description: 'Power Platform user name to authenticate with, e.g. myname@my-org.onmicrosoft.com. Setting this input makes user-name and password required; specifying additional app-id authentication inputs will result in an error.'
```

Comment:
now down to nit-picking phrasing :-)
instead of: "... specifying additional app-id..." -> ...specifying the alternate 'app-id' credential set....

and similar change to the app-id description as well

Reply:
got it.